cmake/config/CMakeLists.txt

```diff
@@ -64,56 +65,22 @@ install(FILES ${_macros}
 #                                                                      #
 ########################################################################
 
-#
-# Strip -Wno-deprecated-declarations from DEAL_II_CXX_FLAGS so that
-# deprecation warnings are actually shown for user code:
-#
-
-strip_flag(DEAL_II_CXX_FLAGS "-Wno-deprecated-declarations")
-
-#
-# Strip -Wsuggest-override. Enabling this warning is a stylistic choice
-# that might create a substantial amount of warnings in user codebases.
-# Let's be nice and disable the flag for now.
-#
-
-strip_flag(DEAL_II_CXX_FLAGS "-Wsuggest-override")
-
```

Comment:
What happened with these flags? AFAICT, this pull request doesn't try to introduce extra warning flags, yet.

Reply:
Yes, I absolutely agree with that strategy and yes, we should do that later. 
Let's hope that we don't break all kinds of tests for builds using `-Werror`.